akka-docs-dev/rst/scala/http/routing-dsl/directives/security-directives/index.rst

```diff
@@ -64,3 +64,18 @@ they are only a means of extracting the so called ``Bearer Token`` from the ``Au
 as defined in `RFC 6750`_, and allow users to validate and complete the protocol.
 
 .. _RFC 6750: https://tools.ietf.org/html/rfc6750
+
+
+.. _credentials-and-timing-attacks-scala:
+
+Credentials and password timing attacks
+---------------------------------------
+
+When transforming request `Credentials` into an application specific user identifier the naive solution for
+checking the secret (password) would be a regular string comparison, but doing this would open up the application to
+timing attacks. See for example http://emerose.com/timing-attacks-explained for an explanation of the problem.
```

Comment:
Having that said... maybe do make it have a title; we usually do links as `Blabla`_ AFAIR. "It is mroe explained in [nice human readable title, not url]"